### PR TITLE
feat: Update actions.yml within the workflow

### PR DESCRIPTION
**Pull request by damacus**
This will update actions.yml Docker image to point at the newly built
image

Signed-off-by: Dan Webb <dan.webb@damacus.io>

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ inputs:
     required: false
 runs:
   using: "docker"
-  image: docker://ghcr.io/actionshub/publish-gem-to-github:v1.0.0
+  image: docker://ghcr.io/actionshub/publish-gem-to-github:pr-10
```
Comment (xorima) [inline action.yml:19]: This looks wrong mate

**Review by xorima — DISMISSED** [review]

File: action.yml (hunk 1 above)
Comment (damacus) [reply action.yml:19]: This is only for the PR. It gets fixed on main again.

**Comment by damacus** [comment]
The docker image version should get fixed on main when the PR runs now. If it doesn't then I need to rethink this later.

**Review by xorima — APPROVED** [review]

**Comment by xorima** [comment]
Approved - my screen was doing weird things on the pr tag

**Pull request merged**